src/main/java/uk/gov/register/views/EntryProof.java

```diff
@@ -31,7 +31,7 @@ public String getEntryNumber() {
 
     @SuppressWarnings("unused, used as jsonproperty")
     @JsonProperty("merkle-audit-path")
-    public List<String> getAuditPath() {
+    public List<HashValue> getAuditPath() {
```

Comment:
Please also update either `VerifiableLogResourceTest` or `VerifiableLogResourceFunctionalTest` to add a test to make sure the correct prefix is added to the hashes in the `merkle-audit-path`.